### DISCUSSION TITLE
Custom Docker image

### DISCUSSION THREAD

**Discussion by iamshoXy** [WeblateOrg/weblate#11330] [Q&A]
I would like to make some modifications to the Weblate repo. changed the link to the fork here: https://github.com/WeblateOrg/docker/blob/5726b11e46a1820511cf341fc557e7396263f175/Dockerfile#L121

Built the image and changed the image name in docker-compose-https.yml

It's still using the official repo, what am I missing?

**Comment by nijel** [comment]
You need to specify `-f docker-compose-https.yml` to `docker compose` to use that file (or rename it to `docker-compose.yml`).

**Comment by iamshoXy** [comment]
That's what I am using: `docker compose -f docker-compose-https.yml -f docker-compose-https.override.yml up -d`

**Reply by nijel** [reply]
Maybe the override specifies the official image still? But I have no clue what content is in your files so I'm only guessing.

**Reply by iamshoXy** [reply]
For the override I am just setting some variables such as the email server and the admin account.
`docker ps` returns the correct image though, I wonder if that's a caching issue.
